MoreLinq/PublicAPI/netstandard2.1/PublicAPI.Shipped.txt

```diff
@@ -429,7 +429,6 @@ static MoreLinq.MoreEnumerable.Cartesian<T1, T2, T3, TResult>(this System.Collec
 static MoreLinq.MoreEnumerable.Cartesian<T1, T2, TResult>(this System.Collections.Generic.IEnumerable<T1>! first, System.Collections.Generic.IEnumerable<T2>! second, System.Func<T1, T2, TResult>! resultSelector) -> System.Collections.Generic.IEnumerable<TResult>!
 static MoreLinq.MoreEnumerable.Choose<T, TResult>(this System.Collections.Generic.IEnumerable<T>! source, System.Func<T, (bool, TResult)>! chooser) -> System.Collections.Generic.IEnumerable<TResult>!
 static MoreLinq.MoreEnumerable.CompareCount<TFirst, TSecond>(this System.Collections.Generic.IEnumerable<TFirst>! first, System.Collections.Generic.IEnumerable<TSecond>! second) -> int
-static MoreLinq.MoreEnumerable.Concat<T>(this System.Collections.Generic.IEnumerable<T>! head, T tail) -> System.Collections.Generic.IEnumerable<T>!
```

Comment:
At the very least, these lines should not be removed. Instead a *REMOVED* version of the line should be added to the unshipped txt file. 

https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md#rs0017-remove-deleted-types-and-members-from-the-declared-api

Reply:
Thank you that one is resolved now. There's a new issue that I'll describe in a separate comment!